builder-support/debian/authoritative/debian-buster/pdns-server.lintian-overrides

```diff
@@ -1,4 +1,2 @@
 # Often this file will contain passwords.
-pdns-server: non-standard-file-perm etc/powerdns/pdns.conf 0600 != 0644
-# We load lsb-functions conditionally.
-pdns-server: init.d-script-needs-depends-on-lsb-base
+pdns-server: non-standard-file-perm etc/powerdns/pdns.conf 0640 != 0644
```

Comment:
0640 is a fine permission to have though?

Reply:
yes :) it just complains because "non-standard"